src/server/snap.rs

```diff
@@ -269,6 +271,7 @@ impl RecvSnapContext {
         if let Err(e) = raft_router.send_raft_msg(self.raft_msg) {
             return Err(box_err!("{} failed to send snapshot to raft: {}", key, e));
         }
+        info!("saving all snapshot files";"snap_key" => %key,"duration" => ?self.start.saturating_elapsed());
```

Comment:
```suggestion
        info!("saving all snapshot files";"snap_key" => %key,"taks" => ?self.start.saturating_elapsed());
```
BTW, Do you run `cargo fmt`?